syntax = "proto3";


package proto_renderer;

import "svg.proto";

message Coordinates{
	double lat = 1;
	double lng = 2;
}
message RenderSettings{
	double width = 1;
	double height = 2;
	double padding = 3;
	double line_width = 4;
	double stop_radius = 5;
	int32 bus_label_font_size = 6;
	proto_svg.Point bus_label_offset = 7;
	int32 stop_label_font_size = 8;
	proto_svg.Point stop_label_offset = 9;
	proto_svg.Color underlayer_color = 10;
	double underlayer_width = 11;
	repeated proto_svg.Color color_palette = 12;
}


message MapRenderer{
	repeated Coordinates stops_coords = 4;
	RenderSettings render_settings = 5;
}